### PR TITLE
fix(slack): add `instillSecret: true` to `token` field

### PR DESCRIPTION
**Pull request by chuang8511**
Because

- We want the token is secret.

This commit

- make the token secret

### CODE REVIEW

File: pkg/connector/slack/v0/config/definition.json [hunk 1]
```diff
@@ -23,12 +23,13 @@
               "string"
             ],
             "instillCredentialField": false,
```
Comment (donch1989) [inline pkg/connector/slack/v0/config/definition.json:25]: Please remove this line.

File: pkg/connector/slack/v0/config/definition.json [hunk 2]
```diff
@@ -23,12 +23,13 @@
...
+            "instillSecret": true,
             "instillUIOrder": 0,
             "title": "token",
             "type": "string"
           }
         },
-        "required": [],
+        "required": ["token"],
         "title": "Slack Connection",
```
Comment (donch1989) [inline pkg/connector/slack/v0/config/definition.json:33]: Add additional `"instillEditOnNodeFields": ["token"]` to make it appears on the Console node.

**Review by donch1989 — CHANGES_REQUESTED** [review]

**Comment by donch1989** [comment]
Found another bug: if the conversations array length is 0, it will return nil, but we should return an empty array. You can try make the start_date larger then now.

**Comment by donch1989** [comment]
Please add `"instillUIMultiline": true` to `message` in TASK_WRITE_MESSAGE

**Comment by chuang8511** [comment]
> Please add `"instillUIMultiline": true` to `message` in TASK_WRITE_MESSAGE
It is originally true in TASK_WRITE_MESSAGE

image

**Comment by donch1989** [comment]
> > Please add `"instillUIMultiline": true` to `message` in TASK_WRITE_MESSAGE
> 
> @donch1989 It is originally true in TASK_WRITE_MESSAGE
> 
> image

Oh... Okay. My eyes are blurred.

**Review by donch1989 — APPROVED** [review]

**Pull request merged**